cstore_fdw.h

```diff
@@ -170,6 +170,15 @@ typedef struct StripeSkipList
 } StripeSkipList;
 
 
+typedef struct BlockDataBuffers
```

Comment:
ColumnBlockDataBuffers is getting too long. we will have problems fitting into single line when we have statement like
ColumnBlockDataBuffers *columnBlockDataBuffers = columnDataBuffers->blockDataBufferArray[blockIndex];

this is 101 character long if I haven't miscounted. 

could we shorten them a bit ?